scripts/validate_docstrings.py

```diff
@@ -531,6 +537,13 @@ def validate_one(func_name):
         examples_errs = doc.examples_errors
         if examples_errs:
             errs.append('Examples do not pass tests')
+        examples_source_code = doc.examples_source_code
+        if 'import numpy' in ' '.join(examples_source_code):
+            errs.append("Numpy does not need to be imported in the examples, "
+                        "as it's assumed to be already imported as np")
+        if 'import pandas' in ' '.join(examples_source_code):
+            errs.append("Pandas does not need to be imported in the examples, "
```

Comment:
I think we don't usually use capitalized pandas, even at the beginning of sentence. And in this case, as it's more the name of the module than the project, I'd use lower case also for numpy.